Sources/Bow/Data/Option.swift

```diff
@@ -283,6 +283,13 @@ extension ForOption: Semigroupal {
 	}
 }
 
+// MARK: Instance of `Monoidal` for `Option`,
```

Comment:
Could you indent correctly the `ForOption` in `Semigroupal`? - I know it is from different PR :)